validation/jinja2/validation/segment_edit.html

```diff
@@ -4,6 +4,6 @@ <h5>Edit Segment</h5>
     <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
     {{ form }}
     <input data-cy="save-button" type="submit" class="button button--success" value="Save">
-    <input data-cy="cancel-button" type="button" class="button button--fail" onclick="hideDiv()" value="Cancel">
+    <input data-cy="cancel-button" type="button" class="button button--fail" value="Cancel">
```

Comment:
Should this be a `<button>`?

```suggestion
    <button type="button" data-cy="cancel-button" type="button" class="button button--fail">Cancel</button>
```